docs/Makefile

```diff
@@ -157,3 +158,12 @@ coverage:
 	$(SPHINXBUILD) -b coverage $(ALLSPHINXOPTS) $(BUILDDIR)/coverage
 	@echo "Testing of coverage in the sources finished, look at the " \
 	      "results in $(BUILDDIR)/coverage/python.txt."
+
+genkeyimg:
+	mkdir -p ./_static/keybindings
```

Comment:
We have both a .gitkeep for this and this mkdir -p; can we just get rid of the gitkeep?

Reply:
removed .gitkeep